### DISCUSSION TITLE
Auto fall back to the date/time display?

### DISCUSSION THREAD

**Discussion by o0shojo0o** [pixelit-project/PixelIt#54] [Ideas]
Meine Überlegung ist das man sagen könnte das wenn nach einer konfigurierten zeit kein neuer Screen an der PixelIt ankommt das diesen dann wieder auf die Datum/Uhrzeit Anzeige zurückspringt. 
Ich selbst brauche diese Funktion nicht, daher die Frage an Euch, braucht man das? 

---
My thought is that you could say that if after a configured time no new screen arrives at the PixelIt that this then jumps back to the date / time display. 
I myself do not need this function, so the question to you, does one need that?

**Comment by foorschtbar** [comment]
Find ich gut. Da ich bei mir ja MQTT benutze, wollte ich auch mal anzeigen, dass die Verbindung zum Broker unterbrochen wurde. Da wäre ich aber auf einen "Connection lost Screen" gegangen￼￼. Hatte ich auch mal zum Testen in der Firmware, aber irgendwie hat es mit der Scroll Geschwindigkeit auf Anhieb nicht geklappt, da hab ich das wieder rausgeschmissen￼￼. 

Aber ja, Fallback auf Uhrzeit als Default an Option fände ich gut

**Reply by o0shojo0o** [reply]
> Da wäre ich aber auf einen "Connection lost Screen" gegangen

Das könnte man dann im Stepp 2 machen, dass man konfigurieren kann was kommen soll wenn kein Screen mehr eintrifft.

**Reply by foorschtbar** [reply]
Das sind zwei Baustellen, sry das ich dass zusammen geworfen habe. Bei MQTT bekommst du ja durch das keepAlive nen echte "no Connection to broker" und nicht nur "da kommen seit 5min keinen screens mehr an"

**Comment by rliegmann** [comment]
Ich finde ich auch gut. Ich muss ehrlich sagen, dass ich bis jetzt keine Verbindungsabbrüche zwischen nodered und PixelIT hatte :-)
Man könnte auch eventuell den letzten Screen lassen und z.B. den letzten Pixel auf rot schalten. Dies bedeutet dann, dass keine Daten mehr von Nodered kommen. 
Ich denke aber das die Zeit aufschalten am sinnvollsten ist.

**Comment by o0shojo0o** [comment]
Das Issues dazu #57

**Reply by o0shojo0o** [reply]
Ist in der aktuellen Repo Version umgesetzt.
